src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -296,9 +296,6 @@ export default compose(
         network: {
             key: ONYXKEYS.NETWORK,
         },
-        currentlyViewedReportID: {
-            key: ONYXKEYS.CURRENTLY_VIEWED_REPORTID,
```

Comment:
Rajat pointed out in 1:1 that `currentlyViewedReportID` both in Onyx and state is confusing.

@mdneyazahmad when using a single tab, the Onyx key `currentlyViewedReportID` is in sync with the state.
But when using multiple tabs, this Onyx key has the value of the most recently viewed report.

What do you think of renaming this key to something like `MOST_RECENTLY_VIEWED_REPORTID`.

Reply:
I think  renaming `currentlyViewedReportID ` to `mostRecentlyViewedReportID` in `onyx` and `currentlyViewedReportID` is fine in state. What do you think?

Reply:
We should be adding ["rename" migration](https://github.com/Expensify/App/blob/main/src/libs/migrations/RenamePriorityModeKey.js) anytime we change an Onyx key so the old value transfers to the new key name. But maybe not worth the effort in this case...

Reply:
Thanks! We'll get to this once performance issue is addressed.